qp_shotgun/humann2/__init__.py

```diff
@@ -0,0 +1,13 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from .humann2 import (
```

Comment:
Are these functions tailored to be used outside of the humann2 module? IMO, it looks like they're internal functions. If so, you don't need to export them here, and then just add one more `.humann2` in the test import.